pycheribuild/qemu_utils.py

```diff
@@ -108,7 +109,9 @@ def disk_image_args(self, image: Path, image_format: str) -> list:
 
         if self.virtio_disk:
             # RISC-V doesn't support virtio-blk-pci, we have to use virtio-blk-device
-            device_kind = "virtio-blk-device" if self.xtarget.is_riscv(include_purecap=True) else "virtio-blk-pci"
+            device_kind = "virtio-blk-device" if self.xtarget.is_riscv(include_purecap=True) or \
+                                                 self.force_virtio_blk_device                   \
+                                              else "virtio-blk-pci"
```

Comment:
Not keen on backslash line continuations, maybe just convert it to an if?
```suggestion
            if self.xtarget.is_riscv(include_purecap=True) or self.force_virtio_blk_device:
                device_kind = "virtio-blk-device"
            else:
                device_kind = "virtio-blk-pci"
```

Although I think we can hopefully drop the RISC-V special case soon? @jrtc27 ?

In that case the following should look better:

```suggestion
            device_kind = "virtio-blk-device" if self.force_virtio_blk_device else "virtio-blk-pci"
```

Reply:
Yes, fixed as of CTSRD-CHERI/cheribsd@fb7c6d17d9dedc30b2ad7ea4ba186c966fa756e8